plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogConnectorSmokeTest.java

```diff
@@ -68,7 +77,8 @@ protected QueryRunner createQueryRunner()
                 .setIcebergProperties(
                         ImmutableMap.of(
                                 "iceberg.catalog.type", "glue",
-                                "hive.metastore.glue.default-warehouse-dir", schemaPath()))
+                                "hive.metastore.glue.default-warehouse-dir", schemaPath(),
+                                REGISTER_TABLE_PROCEDURE_ENABLED, "true"))
```

Comment:
Use `"iceberg.register-table-procedure.enabled"` instead.
In case that the constant  you are referring would accidentally change, the test wouldn't notice the change.

Reply:
Got the intention behind using a string literal than string constant variable. Thanks!